openwhisk/filetype.go

```diff
@@ -65,3 +65,13 @@ func IsZip(buf []byte) bool {
 		(buf[2] == 0x3 || buf[2] == 0x5 || buf[2] == 0x7) &&
 		(buf[3] == 0x4 || buf[3] == 0x6 || buf[3] == 0x8)
 }
+
+// IsTarGz checks if the given file is a valid tar.gz file
+func IsTarGz(buf []byte) bool {
```

Comment:
I think this is only checking for a gzip file.  Is there a check for tar format too?  or should the name of the function be changed to`isGz`

Reply:
Added your suggestions 👍